forge-gui/res/cardsfolder/upcoming/hexavus.txt

```diff
@@ -4,7 +4,7 @@ Types:Artifact Creature Construct
 PT:0/0
 K:etbCounter:P1P1:6
 K:Flying
-A:AB$ PutCounter | Cost$ 1 SubCounter<1/P1P1> | CounterType$ Flying | CounterNum$ 1 | ValidTgts$ Creature.Other | TgtPrompt$ Select target creature | SpellDescription$ Put a flying counter on another target creature.
+A:AB$ PutCounter | Cost$ 1 SubCounter<1/P1P1> | CounterType$ Flying | CounterNum$ 1 | ValidTgts$ Creature.Other | TgtPrompt$ Select another target creature | SpellDescription$ Put a flying counter on another target creature.
 A:AB$ PutCounter | Cost$ 1 RemoveAnyCounter<1/Any/Creature.YouCtrl+Other/another creature you control> | CounterType$ P1P1 | CounterNum$ 1 | SpellDescription$ Put a +1/+1 counter on CARDNAME.
 DeckHas:Ability$Counters
```

Comment:
probably also `DeckHints:Ability$Counters` since this can synergize so well with other cards that make counters